include/game/bosses/boss_9.h

```diff
@@ -5,6 +5,12 @@ extern void CreateTrueArea53Boss(void);
 extern bool32 ExtraBossIsDead(void);
 extern void sub_804D594(s32, s32);
 
+typedef struct {
```

Comment:
Maybe add a TODO here to move this back to `boss_9.c` once we have decompiled it :)